aws_lambda_powertools/logging/formatters/datadog.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from aws_lambda_powertools.logging.formatter import LambdaPowertoolsFormatter
+
+
+class DatadogLogFormatter(LambdaPowertoolsFormatter):
+    def __init__(
```

Comment:
Can we add some docstring explaining why this exist and what the small difference is to the base class?

Reply:
you're the best! Doing that now!

Reply:
Done. Added a specific list so we can easily append if future changes are required - lemme know if that addresses it.